### DISCUSSION TITLE
关于舵机代码里的问题

### DISCUSSION THREAD

**Discussion by yangxiaofeng5** [peng-zhihui/ElectronBot#105] [Q&A]
在舵机的工程代码里，我重新生成了一遍工程，然后编译，发现有一个Main()函数，“空有其名”，不知道是我没找到函数主体还是说什么其它原因，求解答！，谢谢！

**Comment by Kitem** [comment]
您好，我是许焕挺，已收到您的来信，谢谢。

**Comment by yangxiaofeng5** [comment]
编译后，main.c和userApp里面的main.cpp会报错
